php-nginx/etc/confd/templates/postfix/main.cf.tmpl

```diff
@@ -0,0 +1,24 @@
+append_dot_mydomain = no
+biff = no
+header_size_limit = 4096000
+inet_interfaces = loopback-only
```

Comment:
IPv6 DNS resolution on Amazon was a problem for us, so we had to use IPv4 only. Fix from @fratcliffe incoming, though fixing IPv6 DNS might be the best option.